spec/controllers/step_by_step_pages_controller_spec.rb

```diff
@@ -204,9 +204,19 @@ def stub_publishing_api_for_scheduling
       .and_return(content_item(step_by_step_page))
   end
 
-  def schedule_publishing(step_by_step_page, schedule_time = (Time.zone.now + 1.day).to_s)
+  def schedule_publishing(step_by_step_page, schedule = nil)
     stub_default_publishing_api_put_intent
-    post :schedule, params: { step_by_step_page_id: step_by_step_page.id, scheduled_at: schedule_time }
+    if !schedule
```

Comment:
Rather than adding a conditional here, the convention is to add default parameters to the method definition. 

e.g. `def schedule_publishing(step_by_step_page, year: 1.year.from_now.year, month: 1.year.from_now.month, day: 1.year.from_now.day, time: '9:00am')`

By using named parameters, the calling method can substitute any or all of the default parameters.

An interesting fact is that you don't need to use "tomorrow" to get a year and a day from now, as `from_now` seems to do that anyway.
See:  https://stackoverflow.com/questions/39586380/1-year-from-now-is-also-one-day-ahead?rq=1

Reply:
As this method is only be used in two places, I wonder if it's worth keeping? I think it makes the tests more complicated. 

`stub_default_publishing_api_put_intent` could just be moved to the `before` block.

Reply:
Hmmm I think having this helper method is nicer than duplicating the `stub_default_publishing_api_put_intent`, `post :schedule, params ...` and `step_by_step_page.reload` in two places. Let me know what you think once I've pushed up the latest rebase 👍

Reply:
To be fair, the `post` method is the thing being tested, so it would be nice to be able to clearly see the differences in how it's being called.

`stub_default_publishing_api_put_intent` can be added to the `before` block so it's only called once. 

That only leaves reloading the step by step, and I think it's fine to call that in two places.